tests/JWT.Tests/EncodeTests.cs

```diff
@@ -1,72 +1,73 @@
 ﻿using System.Collections.Generic;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Xunit;
 
 namespace JWT.Tests
 {
-    [TestClass]
     public class EncodeTests
     {
         private static readonly Customer _customer = new Customer { FirstName = "Bob", Age = 37 };
 
         private const string _token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJGaXJzdE5hbWUiOiJCb2IiLCJBZ2UiOjM3fQ.cr0xw8c_HKzhFBMQrseSPGoJ0NPlRp_3BKzP96jwBdY";
```

Comment:
Shouldn't then we use these variables as test's parameters in conjunction with `[Theory]`?